specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
           },
-          {
-            "value": 4,
-            "name": "IntermittentClouds",
-            "description": "Intermittent Clouds"
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/{format}": {
+      "get": {
+        "description": "**Get Tropical Storm Search**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nSearch government-issued tropical storms by year, basin ID, and government ID. Information about the tropical storms includes, government ID, basin ID, status, year, name and if it is subtropical.",
+        "operationId": "Weather_GetTropicalStormSearch",
+        "x-ms-examples": {
+          "Get Tropical Storm Search": {
+            "$ref": "./examples/GetStormSearch.json"
           },
+          "Get Tropical Storm Search With Three Properties": {
+            "$ref": "./examples/GetFullStormSearch.json"
+          }
+        },
+        "parameters": [
           {
-            "value": 5,
-            "name": "HazySunshine",
-            "description": "Hazy Sunshine"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
           {
-            "value": 6,
-            "name": "MostlyCloudy",
-            "description": "Mostly Cloudy"
+            "$ref": "#/parameters/ApiVersion"
           },
           {
-            "value": 7,
-            "name": "Cloudy",
-            "description": "Cloudy"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
           },
           {
-            "value": 8,
-            "name": "Dreary",
-            "description": "Dreary (Overcast)"
+            "$ref": "#/parameters/WeatherYear"
           },
           {
-            "value": 11,
-            "name": "Fog",
-            "description": "Fog"
+            "$ref": "#/parameters/OptionalWeatherBasinId"
           },
           {
-            "value": 12,
-            "name": "Showers",
-            "description": "Showers"
+            "$ref": "#/parameters/OptionalWeatherGovernmentId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormSearchResponse"
+            }
           },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/forecasts/{format}": {
+      "get": {
+        "description": "**Get Tropical Storm Forecasts**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued tropical storm forecasts. Information about the forecasted tropical storms includes, location, status, date the forecast was created, window, wind speed and wind radii.",
```

Comment:
Change:
Get all government-issued tropical storm forecasts.

To:
Get individual government-issued tropical storm forecasts.

Reply:
Updated!